src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const paramsCheck = require('./paramsCheck');
+const getParameters = require('./getParameters');
+const sendResponse = require('./sendResponse');
+
+const createServer = () => {
+  const server = http.createServer((request, response) => {
+    response.setHeader('Content-Type', 'application/json');
+
+    const { text, toCase } = getParameters(request.url, `http://${request.headers.host}`);
+
+    const errorMessages = paramsCheck(text, toCase);
+
+    if (errorMessages.length) {
+      const errorObjects = errorMessages.map(message => ({ message }));
```

Comment:
Just push your errors in `paramsCheck` like` { message: text }` instead do extra iteration here
```suggestion
```